thread/src/components/LoginForm.js

```diff
@@ -1,57 +1,89 @@
 import { Actions } from 'react-native-router-flux';
 import * as actions from '../actions';
-import { LoadingButton } from './Common';
+import styled from 'styled-components';
+import Logo from './Logo';
+import {LoadingButton, CommonContainer, CommonField, CommonPassword} from './Common';
+
+const TermsButtonStyle = styled(Text)`
+  font-size:14px;
+  text-align:center;
+  color:white;
+
+  margin-top:40px;
+`;
+
 
 class LoginForm extends Component {
-    
-    onSubmitPress = () => {
-        const { email, password } = this.props;
 
-        this.props.loginUser({ email, password });
-    }
+  onEmailChange(text) {
+     this.props.loginEmailChanged(text);
+     }
+
+     onPasswordChange(text) {
+         this.props.loginPasswordChanged(text);
+     }
+
+   onSubmitPress = () => {
+       const { email, password } = this.props;
+
+       this.props.loginUser({ email, password });
+   }
 
     render() {
         return (
-            <Container>
-                <Content>
-                    <Form>
-                        <Item stackedLabel>
-                            <Label>Email</Label>
-                            <Input
-                                onChangeText={this.props.loginEmailChanged}
-                                value={this.props.email}
-                            />
-                        </Item>
-                        <Item stackedLabel last>
-                            <Label>Password</Label>
-                            <Input
-                                secureTextEntry
-                                onChangeText={this.props.loginPasswordChanged}
-                                value={this.props.password}
-                            />
-                        </Item>
+
+            <CommonContainer>
+              <Content>
+                <Logo />
+                  <Form>
+                      <CommonField
+                          onChangeText={this.onEmailChange.bind(this)}
+                          value={this.props.email}
+                          style={{marginTop: 30, marginLeft: -10}}
+                          label="Username"
+                          />
+                        <CommonPassword
+                          onChangeText={this.onPasswordChange.bind(this)}
+                          value={this.props.password}
+                          style={{marginTop: 30, marginLeft: -10}}
+                          label="Password"
+                        />
                     </Form>
                     <LoadingButton
-                        loading={this.props.loading}
-                        style={{ marginTop: 25 }}
-                        onPress={this.onSubmitPress}
-                        text="Login"
+                      loading={this.props.loading}
+                      style={styles.buttonStyle}
+                      onPress={this.onSubmitPress.bind(this)}
+                      text="Login"
+                    />
+                    <LoadingButton
```

Comment:
Why two loading buttons? To stop user from registering?

Reply:
So that users don't press register after they click login, it would cause unnecessary issues

Reply:
👍